eng/pipelines/templates/jobs/live.tests.yml

```diff
@@ -217,7 +217,7 @@ jobs:
 
               node ../../../common/scripts/install-run-rushx.js integration-test:$(TestType)
               exit $LASTEXITCODE
-          displayName: "Integration test libraries"
+          displayName: "Integration test libraries - PackagePath"
```

Comment:
I know these get cut off sometimes so it might be worth putting this in the front.

Reply:
The downside if it's in the front is then the UI will show something like "PackagePath -" which is even less helpful. At least this way when you hover/select the steps you can tell which is which.